plugins/outputs/opentsdb/opentsdb_http.go

```diff
@@ -16,7 +16,7 @@ import (
 type HttpMetric struct {
 	Metric    string            `json:"metric"`
 	Timestamp int64             `json:"timestamp"`
-	Value     string            `json:"value"`
+	Value     interface{}       `json:"value"`
```

Comment:
can you explain this? how was this working before if every value was being sent as a string?